onnxruntime/core/providers/shared_library/provider_wrappedtypes.h

```diff
@@ -932,6 +978,10 @@ struct NodeUnit final {
   // output. any Q nodes are hidden.
   Node::EdgeConstIterator OutputEdgesBegin() const { return g_host->NodeUnit__OutputEdgesBegin(this); }
   Node::EdgeConstIterator OutputEdgesEnd() const { return g_host->NodeUnit__OutputEdgesEnd(this); }
+
+  NodeUnit() = delete;
+  NodeUnit(const NodeUnit&) = delete;
+  void operator=(const NodeUnit& v) = delete;
```

Comment:
Why not use PROVIDER_DISALLOW_ALL here?

Reply:
We need to define the delete operator because another API returns a vector of `unique_ptr<NodeUnit>`.

Reply:
Ah, I see it above, maybe it should be grouped with them?

Reply:
Sounds good. I moved them up and added a comment explaining why need delete.